src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecordDto;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitShopService {
```

Comment:
Add in this service a new method that would read data from db and return you String version of your report

Reply:
add interface ReportCreator